docs/integrations/saas-cloud/box.md

```diff
@@ -11,13 +11,11 @@ import useBaseUrl from '@docusaurus/useBaseUrl';
 
 The Sumo Logic App for Box provides insight into user behavior patterns, monitors resources, and even tracks administrative activities. The app consists of three predefined Dashboards, providing visibility into your environment for real time analysis.
 
+## Log types
 
-## Log Types
+The Sumo Logic app for Box collects Box events, which are described in detail [here](https://developers.box.com/docs/#events).
```

Comment:
The URL provided gives a privacy error. Would this be a better URL to use? https://developer.box.com/reference/resources/event/

Reply:
Yes John, the link you suggested will be good. Thanks for your suggestion.